pkg/conversion/converter.go

```diff
@@ -213,6 +213,8 @@ type Meta struct {
 	// KeyNameMapping is an optional function which may map the listed key (field name)
 	// into a source and destination value.
 	KeyNameMapping FieldMappingFunc
+	// Context is an optional field that callers may use to pass info to conversion functions.
+	Context interface{}
```

Comment:
This will tie particular conversions to particular encoders.  I'm not strictly against it since I can see the necessity for converting embedded objects (`Templates` come to mind), but are we sure this is the interface we want instead of forcing a little more structure and sending a `GroupVersioner`.